python/GafferArnoldUI/ArnoldAttributesUI.py

```diff
@@ -304,6 +304,110 @@ def __toonSummary( plug ) :
 
 		],
 
+		"attributes.cameraAutobumpVisibility" : [
+
+			"description",
+			"""
+			Whether or not the autobump is visible to camera
+			rays.
+			""",
+
+			"layout:section", "Visibility",
+			"label", "Autobump Camera",
+
+		],
+
+		"attributes.shadowAutobumpVisibility" : [
+
+			"description",
+			"""
+			Whether or not the autobump is visible to shadow
+			rays (whether or not it casts shadows).
+			""",
+
+			"layout:section", "Visibility",
+			"label", "Autobump Shadow",
+
+		],
```

Comment:
Is autobump actually useful for shadow rays? @danieldresser-ie ?

Reply:
Summary of various internal conversations : it could potentially be useful in some cases, but it doesn't currently work, so it's best not to expose it for now.